src/styles/main.scss

```diff
@@ -1,3 +1,80 @@
+$paddingCalendar: 10px;
+$gapFeature: 1px;
+$sideOfSquare: 100px;
+$widthOfCalendar: (
+  7 * $sideOfSquare) + (6 * $gapFeature) + (2 * $paddingCalendar
+);
+$marginFromLeftSide: (
+  "mon": 0,
+  "tue": 1,
+  "wed": 2,
+  "thu": 3,
+  "fri": 4,
+  "sat": 5,
+  "sun": 6
+);
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.page {
+  width: 100%;
+  height: 100%;
```

Comment:
<img width="169" alt="Screenshot 2023-10-29 at 15 43 48" src="https://github.com/mate-academy/layout_calendar/assets/135232001/eb1d561b-0323-4d1d-af39-4858bdcf6920">
Add these styles to the page to center your calendar vertically

Reply:
If I used display flex, would it be better to use justify-content and align-items?